### ISSUE TITLE
AI Stock Trading agent to predict stock prices | Live trading

### ISSUE THREAD

**Issue by jinglescode** [jinglescode/jinglescode.github.io#57]
# TL;DR
Alpaca API for live data and perform trades

# Key Takeaways

- **Alpaca Broker account** Currently, most of the brokerage firms offer zero trading fees. However, not all brokerage firms have an API option to trade. Alpaca provides free trading with python API to trade. Once you create an account, you will have paper trading and live trading options.

- **Alpaca API get historical data** One advantage of using Alpaca is you can get historical data from polygon API. 
```
import alpaca_trade_api as tradeapi
api = tradeapi.REST('xxxxxxxx', 'xxxxxxxxxx',base_url='https://paper-api.alpaca.markets', api_version='v2',)
df = api.polygon.historic_agg_v2()
```

# Links
- “24x5 Stock Trading Agent to predict stock prices with Deep Learning with deployment” by Shyam BV https://link.medium.com/2kNuUpb1L5
-  Repo: https://github.com/DarkKnight1991/stock_cnn_blog_pub/
